app/routes/admin/content/translations.js

```diff
@@ -12,5 +14,17 @@ export default Route.extend({
         sourceUrl : '#'
       }
     ];
+  },
+
+  actions: {
+    translationsDownload() {
```

Comment:
Also just noticed. Move this action to the controller -- all the controlling logic should reside there.

Reply:
yeah, I was doubtful about whether it should reside in the route or the controller. I'll create a new controller file and move the logic there.